src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserFactory.java

```diff
@@ -0,0 +1,57 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import lombok.Builder;
+import lombok.Getter;
+
+public final class ForgottenPasswordUserFactory {
```

Comment:
So I'd say this class is unnecessary. The ForgottenPasswordUser could be the base class which has got two factory methods.